SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -161,9 +161,11 @@
   private long saveStateEverySeconds = 60;
 
   @JsonProperty("sentry")
+  @Valid
```

Comment:
The sample/docker singularity runs with both of these as null, started up with no issues